apps/hashdotdev/src/_pages/blog/0014_structured-data-value.mdx

```diff
@@ -0,0 +1,109 @@
+3. domain experts, who understand how the target system functions, and can actually provide the subject matter expertise and model design required.
+
+These different stakeholders and the communication/coordination/iteration loops involved in getting simulation models accurate _and useful_ were frankly killer. With data structuring and availability problems potentially already solved by foundation models (as outlined above), **we believe foundation models' further potential lies in enabling domain experts to bypass programmers and iterate on simulation logic directly.** Behaviors within simulations are generally relatively simple, standalone things. Natural language behavior developement seems eminently achievable using current-generation (e.g. GPT-4) models.
+
+The role of structured data in all this is simple.
+
+**Predictive simulations of real-world things are extremely easy to get wrong.** Many interesting systems that users typically want to simulate end up being "complex systems" in some respect, demonstrating some degree of nonlinearity due to emergent or adaptive behaviors.
+
+Complex systems are typically extremely sensitive to their initial conditions. Ensuring an as close to accurate starting or "initial state" is therefore key. Pre-structured data, especially if itself tested and reviewed, or produced in accordance with a well-understood process or pipeline, can improve confidence in this.
+
+In addition to providing the initial state of a simulation, structured data can be used directly for verification and validation purposes, including as a means of backtesting simulation experiments, and individual behaviors themselves. In a world where behaviors and whole simulation models may be generated by foundation models, this provides humans with an improved sense of their likely correctness.
+
+## More for humans than machines
+
+All this serves to highlight an interesting possibility. If to date structured data has existed for the benefit of _machines_, is it possible that in the future its primary use will in fact me to us, _humans_, in verifying and validating the outputs of machine-generated models: simulation, language, multimodal, or other?
+
+Inherently tied to the idea of structured data are _types_. Data may be of a certain 'type' if it follows a set of rules established by that type.
+
+One way of thinking about structured data is the [Block Protocol type system](https://blockprotocol.org/docs/types), which outlines three different kinds of types: data types, property types, and entity types.
+
+1. **Entites** (semantic 'things' in a database) have entity types.
+  - **Entity types** describe the _properties_ and type(s) of any outgoing _link entities_ that can be associated with an entity.
+  - **Link entities** are the relationships, or 'things' that connect other entities. For example, one `Person` entity may be connected by a linked entity called `Spouse` to another `Person`.
```

Comment:
Or possibly `Marriage` – `Spouse` describes the person rather than the relationship
```suggestion
  - **Link entities** are the relationships, or 'things' that connect other entities. For example, one `Person` entity may be connected by a link entity called `Has Spouse` to another `Person`.
```